tests/python/YapfBearTest.py

```diff
@@ -22,6 +24,15 @@ def test_valid(self):
                             ["x = {  'a':37,'b':42,\n", "'c':927}\n", '\n',
                              "y = 'hello ''world'\n"], valid=False)
 
+    def test_valid_python_2(self):
+        self.check_validity(self.uut, ['print 1\n'], valid=True)
+
+    @skipIf(sys.version_info < (3, 5), "async def can't be parsed without 3.5+")
```

Comment:
Also possibly make a test for python 2 syntax please ?
As that was the case mentioned in the issue and that can also be verified for all python versions